docs/zh/client/java-sdk-user-guide.md

```
@@ -1312,3 +1433,274 @@
+
+
+这里引入了skyWalking的micrometer依赖
+```xml
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>apm-toolkit-micrometer-1.10</artifactId>
+        </dependency>
+```
+根据Micrometer的机制初始化SkywalkingMeterRegistry，以及一些map用于存储指标数据
+```java
+	private static final String SKYWALKING = "skywalking";
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)